test.js

```diff
@@ -262,6 +262,79 @@ test('Expose custom status route', t => {
   })
 })
 
+test('Custom health check', t => {
+  t.plan(2)
+
+  t.test('should return 503 when custom health check returns false for healthCheck', t => {
+    t.plan(5)
+
+    const fastify = Fastify()
+    fastify.register(underPressure, {
+      healthCheck: async () => {
+        return false
+      }
+    })
+
+    fastify.get('/', (req, reply) => {
+      reply.send({ hello: 'world' })
+    })
+
+    fastify.listen(0, (err, address) => {
+      t.error(err)
+      fastify.server.unref()
+
+      // Increased to prevent Travis to fail
+      process.nextTick(() => sleep(1000))
```

Comment:
this is a code smell, why is this needed?

Reply:
copy pasta from  previous tests. will remove.